docs/antrea-ipam.md

```diff
@@ -51,11 +51,9 @@ Below is a sample of needed changes in the Antrea deployment YAML:
 
 ```yaml
   antrea-controller.conf: |
-    ...
```

Comment:
It is not a big deal, but no way to express "..."?

Reply:
I don't know a good way. And a ConfigMap including this content only is also correct. The options that are not displayed will just use their defaults. Having "..." in-between the configurations will just make it harder to copy and paste, e.g. https://github.com/antrea-io/antrea/blob/main/docs/antrea-ipam.md#enable-antreaipam-feature-gate-and-bridging-mode

Reply:
Ok. We can remove that.